CODEOWNERS

```diff
@@ -307,9 +307,8 @@ Makefile* @cilium/build
 /Documentation/check-examples.sh @cilium/docs-structure
 /Documentation/check-helmvalues.sh @cilium/docs-structure
 /Documentation/cmdref/
+/Documentation/community/governance.rst @cilium/contributing
```

Comment:
Apologies, I missed this the first time round - I believe we try to keep these in alphabetical order, hence `community/governance` should come after `community/community`.

Reply:
No worries! Fixed it